### ISSUE TITLE
Windows / Python 3.9 not working

### ISSUE THREAD

**Issue by ericfrederich** [zeromq/pyzmq#1459]
The following works if you use `-3.8` instead of `-3.9`

```
C:\Users\ericf>cd %TEMP%

C:\Users\ericf\AppData\Local\Temp>py -3.9 -m venv zmq_env

C:\Users\ericf\AppData\Local\Temp>zmq_env\Scripts\activate

(zmq_env) C:\Users\ericf\AppData\Local\Temp>pip install --upgrade pip wheel
Collecting pip
  Using cached pip-20.3-py2.py3-none-any.whl (1.5 MB)
Collecting wheel
  Using cached wheel-0.36.0-py2.py3-none-any.whl (34 kB)
Installing collected packages: pip, wheel
  Attempting uninstall: pip
    Found existing installation: pip 20.2.3
    Uninstalling pip-20.2.3:
      Successfully uninstalled pip-20.2.3
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'C:\\Users\\ericf\\AppData\\Local\\Temp\\pip-uninstall-l68ui8yb\\pip.exe'
Consider using the `--user` option or check the permissions.


(zmq_env) C:\Users\ericf\AppData\Local\Temp>pip install pyzmq
Collecting pyzmq
  Using cached pyzmq-20.0.0-cp39-cp39-win_amd64.whl (699 kB)
Installing collected packages: pyzmq
Successfully installed pyzmq-20.0.0

(zmq_env) C:\Users\ericf\AppData\Local\Temp>py -c "import zmq"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\ericf\AppData\Local\Temp\zmq_env\lib\site-packages\zmq\__init__.py", line 50, in <module>
    _load_libzmq()
  File "C:\Users\ericf\AppData\Local\Temp\zmq_env\lib\site-packages\zmq\__init__.py", line 28, in _load_libzmq
    from . import libzmq
ImportError: DLL load failed while importing libzmq: The specified module could not be found.
```

**Comment by minrk** [comment]
Thanks for the report! How have you installed Python 3.9? Does `C:\Users\ericf\AppData\Local\Temp\zmq_env\lib\site-packages\zmq\libzmq.cp39-win_amd64.pyd` exist?

Can you show the output of:

```python
import sysconfig
print(sysconfig.get_config_var('EXT_SUFFIX'))
print(sysconfig.get_config_var('SOABI'))
```

**Comment by anch0vy** [comment]
problem is pyzmq with python3.9 wheel is served without "concrt140.dll", "msvcp140.dll".

**Comment by minrk** [comment]
This is fixed in pyzmq 21 by #1470, though there's another issue that will be fixed shortly in 21.0.1 (#1473). Should be out later today.
